docs/community/contribute.md

```diff
@@ -62,13 +62,37 @@ To join the MXNet slack channel send request to the contributor mailing list.
 
 ### Social Media
 
-Keep connected with the latest MXNet news and updates on [Twitter](https://twitter.com/apachemxnet) and [Reddit](https://reddit.com/r/mxnet). Also, subscribe to the [MXNet YouTube channel](https://www.youtube.com/channel/UCQua2ZAkbr_Shsgfk1LCy6A).
+Keep connected with the latest MXNet news and updates.
```

Comment:
This whole section renders bad, with the logos not aligned properly. Needs some work.

Reply:
I started with the community page and hated how ugly that social section was. That's why I added the nav bar. I'll circle back and fix it.

Also I can add higher resolution images. I might need to requestion a 4k monitor 😋

Reply:
Cool.
> I might need to requestion a 4k monitor 😋

I'm sure you can find a manager at AWS that will approve such a request...